action/action-util/src/main/java/com/powsybl/action/util/StackScalable.java

```diff
@@ -25,15 +25,16 @@
         super(scalables);
     }
 
+
     @Override
-    public double scale(Network n, double asked) {
+    public double scale(Network n, double asked, ScalingPowerConvention scalingConvention) {
         Objects.requireNonNull(n);
 
         double done = 0;
         double remaining = asked;
         for (Scalable scalable : scalables) {
-            if (remaining > 0) {
-                double v = scalable.scale(n, remaining);
+            if (Math.abs(remaining) > 0) {
```

Comment:
is it equivalent to `remaining != 0` ?

Reply:
Following this example (http://www.java67.com/2015/09/float-and-double-value-comparison-in-java-use-relational.html) I think we should use an EPSILON threshold don't you think ?

Reply:
Sure! It’s better with an epsilon